src/assets/pages/config.js

```diff
@@ -0,0 +1 @@
+export const Server = 'http://localhost:3000';
```

Comment:
I would suggest to directly use this URL from backend. Try searching about env_info